app/views/users/webauthn_setup/new.html.slim

```diff
@@ -2,6 +2,7 @@
 - help_link = link_to t('links.what_is_webauthn'),
               MarketingSite.help_hardware_security_key_url, target: :_blank
 
+= image_tag asset_url('security-key.svg'), width: '90', class: 'ml1'
```

Comment:
`width: '110'` `class: 'mb2` is what I came up with for this

![screen shot 2018-11-07 at 11 59 22 am](https://user-images.githubusercontent.com/776987/48150613-c3014180-e284-11e8-89ee-29e9f774f825.png)

Reply:
Looks reasonable to me. What do you think, @rtwell ?